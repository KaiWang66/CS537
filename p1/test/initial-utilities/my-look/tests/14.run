./my-look $(< tests/14.target)
