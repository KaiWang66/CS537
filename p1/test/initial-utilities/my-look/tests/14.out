Dada
Dada's
Dadaism
Dadaism's
dad
dad's
daddies
daddy
daddy's
dado
dado's
dadoes
dados
dads
